### PR TITLE
Set active status && create user handler

### PR DESCRIPTION
**Pull request by gcolotti**

### CODE REVIEW

File: src/cloud/setActiveStatus.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import Parse from '../providers/ParseProvider';
+import db from '../utils/db';
+
+class SetActiveStatusError extends ExtendableError {}
+
+/**
+ *
+ * @param {*} user
+ */
+const getUserHandler = async user => {
```
Comment (santiagosemhan) [inline src/cloud/setActiveStatus.js:11]: should be handle instead of handler?

File: src/cloud/setActiveStatus.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  const config = await Parse.Config.get({ useMasterKey: true });
+  const maxQuePosition = config.get('maxQuePosition');
+  // If the user has a quePosition already, use it. Else, get a new quePosition
+  let quePosition;
+  if (user.get('quePosition')) {
+    quePosition = user.get('quePosition');
+  } else {
+    quePosition = await db.getValueForNextSequence('unclaimedPosition');
+  }
+  const handlerPositioN = quePosition / maxQuePosition;
```
Comment (santiagosemhan) [inline src/cloud/setActiveStatus.js:21]: handle

**Review by santiagosemhan — CHANGES_REQUESTED** [review]
function name handle

File: src/cloud/setActiveStatus.js (hunk 1 above)
Comment (gcolotti) [reply src/cloud/setActiveStatus.js]: Yes, sent a new commit already

File: src/cloud/setActiveStatus.js [hunk 3]
```diff
@@ -18,14 +18,14 @@ const getUserHandler = async user => {
   } else {
     quePosition = await db.getValueForNextSequence('unclaimedPosition');
```
Comment (santiagosemhan) [inline src/cloud/setActiveStatus.js:19]: You will increase the unclaimedPosition when you read. Is this right?

**Review by santiagosemhan — CHANGES_REQUESTED** [review]

**Pull request merged**